docs/content/en/docs/user-guide/event-watcher.md

```diff
@@ -0,0 +1,152 @@
+---
+title: "Event watcher"
+linkTitle: "Event watcher"
+weight: 13
+description: >
+  A helper facility to automatically update files when it finds out a new event.
+---
+
+The only way to upgrade your application with PipeCD is modifying configuration files managed by the Git repositories.
+It brings benefits quite a bit, but it can be painful to manually update them every time in some cases (e.g. continuous deployment to your development environment for debugging, the latest prerelease to the staging environment).
+
+If you're experiencing any of the above pains, Event watcher is for you.
+Event watcher works as a helper facility to seamlessly link CI and CD. This feature lets you automatically update files managed by your Piped when an arbitrary event occurs.
+While it empowers you to build pretty versatile workflows, the canonical use case is that you trigger a new deployment by image updates, package releases, etc.
+
+This guide walks you through configuring Event watcher and how to push an Event.
+
+## Prerequisites
+Before configuring EventWatcher, be sure to grant write permission to the SSH key used by Piped. See [here](/docs/operator-manual/piped/configuring-event-watcher/) for more details.
+
+## Usage
+File updating can be done by registering the latest value corresponding to the Event in the control-plane and comparing it with the current value.
+
+Therefore, you mainly need to:
+1. define which values in which files should be updated when a new Event found.
+1. integrate a step to push an Event to the control-plane using `pipectl` into your CI workflow.
+
+### 1. Defining Events
+Prepare EventWatcher files placed at the `.pipe/` directory at the root of the Git repository.
```

Comment:
nits, 
`Prepare EventWatcher configuration files under the `.pipe/` directory at the root (level) of your Git repository.`